frontends/ol-components/src/components/ThemeProvider/typography.ts

```diff
@@ -147,6 +147,11 @@ const globalSettings: ThemeOptions["typography"] = {
 const component: NonNullable<ThemeOptions["components"]>["MuiTypography"] = {
   defaultProps: {
     variantMapping: {
+      h1: "span",
+      h2: "span",
+      h3: "span",
+      h4: "span",
+      h5: "span",
```

Comment:
This is to prevent accidental future headings.

In general, our `h1...h5` font variant may not correspond to an h1...h5 header, which was the old behavior.